drake/solvers/test/testOptimizationProblem.cpp

```diff
@@ -192,10 +192,88 @@ void gloptipolyConstrainedMinimization() {
   valuecheckMatrix(x.value(), Vector3d(.5, 0, 3), 1e-4);
 }
 
+/**
+ * Test that the eval() method of LinearComplementarityConstraint correctly
+ * returns the slack.
+ */
+void simpleLCPConstraintEval() {
```

Comment:
These could be written on googletest with little pain, even without migrating the rest of the file.  Up to you!